DICOM/src/Microsoft.Health.DeID.SharedLib/Settings/RedactSetting.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib.Settings
+{
+    public class RedactSetting
+    {
+        public bool EnablePartialDatesForRedact { get; set; } = false;
```

Comment:
🟡  **SUGGESTION**

Is `ForRedact` necessary in `RedactSetting` class?
It's clear that this is for **redaction**. 
- `_redactSetting.EnablePartialDatesForRedact`
- `_redactSetting.EnablePartialDates`

Which one do you prefer?

Reply:
The name is stemmed from FHIR config.
![image](https://user-images.githubusercontent.com/57981365/123230961-55c07e80-d50a-11eb-949a-706b1ef47ba8.png)
In FHIR, settings for different methods are together. So, "ForRedact" is essential in FHIR.
Here, maybe keep consistent is better?

Reply:
💡 [Discussion]

This is not related to this PR, but is can zip code list be used globally? 😺

Reply:
They are only valid in US scope